internal/pkg/auth/config_test.go

```diff
@@ -43,8 +43,8 @@ func TestEnsureAuthConfigResource(t *testing.T) {
 		},
 		{
 			name: "enable auth0",
-			initialConfig: config.AuthParams{
-				Auth0: config.Auth0Params{
+			initialConfig: config.Auth{
+				Auth0: config.Auth0Config{
```

Comment:
nit, but since `Auth0Config` is already in the `config` package, should it simply be `Auth0`? Just like `AuthParams` -> `Auth`, it would be `Auth0Params` -> `Auth0`.

Reply:
yeah, can do that